src/main/java/org/sunbird/workallocation/service/AllocationServiceV2.java

```diff
@@ -52,7 +52,7 @@
     @Value("${workorder.index.type}")
     public String workOrderIndexType;
 
-    @Value("${workallocation.index.name}")
+    @Value("${workallocationv2.index.name}")
```

Comment:
you suppose to create new --- instead of replacing previous one ?

Reply:
We have created the new only. It's allocationservicev2